index.html

```diff
@@ -1,14 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
lang 속성은 웹 페이지의 언어를 선언하는데요, 

> lang 속성이란? 
> ​현재 페이지의 텍스트가 어떠한 언어로 작성되어 있는지 표기하여 시각적으로 불편하신 사용자 분들께서
> ​스크린 리더와 같은 보조 수단을 통해 웹을 이용하실 때 lang 속성으로 표기한 언어에 알맞도록 발음이 적용되어
> ​웹을 조금 더 편하게 이용하실 수 있도록 도움을 드릴 수 있는 웹 접근성 관련 속성이라고 하네요 😆

아무래도 ko로 바꾸는게 낫겠죠 ? > <
```suggestion
<html lang="ko">
```